bin/katello-certs-check

```diff
@@ -178,6 +178,20 @@ function check-cert-san () {
     CHECK=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
     if [[ $? == "0" ]]; then
         success
+        printf "Checking if any of Subject Alt Names on certificate matches the Subject CN"
+        SUBJECT_CN=$(openssl x509 -in $CERT_FILE -noout -subject -nameopt multiline|grep commonName|cut -d '=' -f 2|tr -d ' ')
+        DNS_LIST="$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)"
```

Comment:
```suggestion
```

We can already get this info from L#178 (see above suggestion)

Reply:
Thanks for the suggestions. I'll commit the changes.